runtime/lua/vim/lsp.lua

```diff
@@ -588,6 +588,12 @@ function lsp.start_client(config)
       -- process is not alive then the server should exit (see exit notification)
       -- its process.
       processId = uv.getpid();
+      -- Information about the client
+      -- since 3.15.0
+      clientInfo = {
+        name = 'Neovim',
```

Comment:
Maybe double quotes here instead of single.

Reply:
I guess it doesn't matter

Reply:
Fixed.